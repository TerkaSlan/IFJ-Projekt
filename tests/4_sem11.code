/*Should be error becasue of assigment string to int*/

class Main {

	static void run() {
		int i = "adw";
	}
}