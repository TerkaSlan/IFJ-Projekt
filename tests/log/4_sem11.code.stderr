ERROR: Semantic - type - Line: 6 - Assigning of unconvertible types.
